currency.go

```diff
@@ -62,6 +63,7 @@ var currencies = map[string]*Currency{
 	"EGP": {Decimal: ".", Thousand: ",", Code: "EGP", Fraction: 2, Grapheme: "\u00a3", Template: "$1"},
 	"ERN": {Decimal: ".", Thousand: ",", Code: "ERN", Fraction: 2, Grapheme: "Nfk", Template: "1 $"},
 	"ETB": {Decimal: ".", Thousand: ",", Code: "ETB", Fraction: 2, Grapheme: "Br", Template: "1 $"},
+  "ETH": {Decimal: ".", Thousand: ",", Code: "ETH", Fraction: 2, Grapheme: "\u039e", Template: "1$"},
```

Comment:
First time I see "Ξ" being used for ethers. A bit confusing IMO

Reply:
There are many discussions if symbol of ether should be aligned with symbol of Ethereum. 

- https://ethereumsymbol.com/
- https://ethresear.ch/t/proposal-change-ether-currency-symbol-from-to/120